docs/source/reference/check.rst

```diff
@@ -55,3 +56,54 @@ Chainer provides some utilities for testing its functions.
    :nosignatures:
 
    chainer.testing.unary_math_function_unittest
+   chainer.testing.save_and_load
+   chainer.testing.save_and_load_hdf5
+   chainer.testing.save_and_load_npz
+   chainer.testing.get_trainer_with_mock_updater
+
+
+Repeat decorators
+-----------------
+
+These decorators have a decorated test run multiple times
```

Comment:
-`These decorators have a decorated test run multiple times in a single invocation.`
+`These decorators can be used to decorate test to run mulitple times in a single invocation.`